nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -66,6 +66,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The minimal ISO image now uses the `nixos/modules/profiles/minimal.nix` profile.
 
+- `services.peertube` now requires you to specify the file with the secret - `secrets.secretsFile`. Generate it with `openssl rand -hex 32`.
```

Comment:
```suggestion
- `services.peertube` now requires you to specify the secret file `secrets.secretsFile`. It can be generated by running `openssl rand -hex 32`.
```